Library/Formula/mydumper.rb

```diff
@@ -0,0 +1,118 @@
+class Mydumper < Formula
+  desc "How MySQL DBA & support engineer would imagine 'mysqldump' ;-)"
+  homepage "https://launchpad.net/mydumper"
+  url "https://launchpad.net/mydumper/0.6/0.6.2/+download/mydumper-0.6.2.tar.gz"
+  sha256 "fa28563e8967752828954c5d81e26ef50aad9083d50a977bf5733833b23e3330"
+
+  option "without-docs", "Don't build man pages"
+
+  depends_on "cmake" => :build
+  depends_on :python => :build if MacOS.version <= :snow_leopard && build.with?("docs")
+  depends_on "glib"
+  depends_on "mysql"
+  depends_on "pcre"
+  depends_on "pkg-config" => :build
+  depends_on "openssl"
+
+  # This patch allows cmake to find .dylib shared libs in OS X. A bug report has
+  # been filed upstream here: https://bugs.launchpad.net/mydumper/+bug/1517966
+  # It also ignores .a libs because of an issue with glib's static libraries now
+  # being included by default in homebrew.
+  patch :p0, :DATA
+
+  resource "sphinx_rtd_theme" do
```

Comment:
You can remove all the resources and python logic now and just do:
```ruby
depends_on "sphinx-doc" => :build if build.with? "docs"
```
People were using Sphinx for documentation regularly enough but not updating the dependencies, so we decided to reduce replication and just create a `sphinx-doc` formula.